MMM-MBTA.js

```diff
@@ -92,170 +106,283 @@ Module.register("MMM-MBTA", {
+                        symbolCell.className += " commuter"
+                    }
 
-                        preAwayCell.innerHTML = minutes + ":" + seconds;
-                    } else {
-                        preAwayCell.innerHTML = seconds;
+                    if (isNaN(this.stationData[i].routeId) === false) {
+                        symbolCell.className += " bus"
                     }
                 }
-                row.appendChild(preAwayCell);
-            }
 
-            if (this.config.showArrivalTime) {
+                symbolCell.style.cssText = "padding-right: 10px";
+                row.appendChild(symbolCell);
+
+                // Description
+                var descCell = document.createElement("td");
+                descCell.innerHTML = this.stationData[i].tripSign;
+
+                //Change routeId to public route name
+                if (isNaN(this.stationData[i].routeId) === false) {
+                    switch (this.stationData[i].routeId) {
+                        case "741":
+                            descCell.innerHTML += " | SL1";
+                            break;
+                        case "742":
+                            descCell.innerHTML += " | SL2";
+                            break;
+                        case "751":
+                            descCell.innerHTML += " | SL4";
+                            break;
+                        case "749":
+                            descCell.innerHTML += " | SL5";
+                            break;
+                        case "701":
+                            descCell.innerHTML += " | CT1";
+                            break;
+                        case "747":
+                            descCell.innerHTML += " | CT2";
+                            break;
+                        case "708":
+                            descCell.innerHTML += " | CT3";
+                            break;
+                        default:
+                            descCell.innerHTML += " | " + routeId;
+                    }
+                }
+                descCell.className = "align-left bright";
+                row.appendChild(descCell);
+        
+                // ETA
+                if (this.config.showETATime) {
+                    var preETACell = document.createElement("td");
+                    var preETATime = this.stationData[i].preETA;
+                    
+                    if (preETATime == null) {
+                        preETACell.innerHTML = "No ETA"
+                    } else if (preETATime < 10) { // Better to display single digits as "now"
+                        preETACell.innerHTML = "Now";
+                    } else {
+                        var minutes = Math.floor(preETATime / 60);
+                        var seconds = preETATime % 60;
+                        
+                        if (this.config.showMinutesOnly) {
+                            if (isNaN(minutes) === true) {
```

Comment:
This also a null check?

Reply:
Yeah. Just replaced it with `if (!minutes)`.